mock/py/mockbuild/util.py

```diff
@@ -1100,7 +1100,7 @@ def setup_default_config_opts(unprivUid, version, pkgpythondir):
     # configurable commands executables
     config_opts['yum_command'] = '/usr/bin/yum'
     config_opts['system_yum_command'] = '/usr/bin/yum'
-    config_opts['yum_install_command'] = 'install yum yum-utils shadow-utils distribution-gpg-keys'
+    config_opts['yum_install_command'] = 'install --disablerepo sclo* yum yum-utils shadow-utils distribution-gpg-keys'
```

Comment:
Could you please put this in the `epel-*` chroot configs instead? You're essentially saying `sclo*` repos are disabled for _all_ targets, and there's more than CentOS and Fedora using mock.

Reply:
Meaning, you don't know what you're disabling by doing this.

Reply:
I may.  Do you have any reasons to keep the sclo* repos enabled anywhere else?

Reply:
Do you know that `sclo*` is going to match only for the Software Collections repos? People can make their own mock configs where those IDs can be anything...

Reply:
Ok, it's rather really theoretical worry which brings slightly more maintenance work (we'll have to think about this when epel-8 very likely, downstream rhel, etc...).   But I don't mind in particular, and I'll move it to epel configs.

Reply:
Thank you.